_posts/2019-07-29-SageDays102.md

```diff
@@ -0,0 +1,434 @@
+creative and interesting interacts. However, it was clear that we needed to
+spend more time in the first half of the day on getting everyone up to speed
+on Sage and Python basics, which was well appreciated.
+
+For the second half of Wednesday we split into four separate, parallel
+sessions for different topic interests:
+
+* Graph theory and discrete math with Sage, headed by Erik and Evans.
+
+* Numerical analysis and calculus with Sage, headed by Yaé.
+
+* Group theory and algebra with GAP, headed by Chimere.
+
+* Statistical analysis with R, headed by Olalekan.
+
+
+![Attendance by session: Graph theory: 11; Calculus: 34; Group theory: 11; Statistics: 18](/public/images/SageDays102/sessions.png){:class="img-responsive"}
+
+It was good to have a representative from the GAP community, in part to
+drive home the point I was making that SageMath could not exist without all
+the technologies it is built on top of, and to give special emphasis to one
+of them.  Similarly with R, although R is included as part of Sage, and Sage
+includes an interface to R, the two are less tightly coupled than Sage and
+GAP.  Nevertheless, although Sage does have useful facilities for
+statistics, it has not often been the focus of most (although some) Sage
+developers.  So this provided some of the best of both worlds for attendees
+who were most interested in statistical analysis.
+
+Chimere also provided a [separate
+report](/public/images/days102-gap-report.pdf) on the workshop and on what
+was taught in his session.  It also includes a breakdown of his total
+expenses for travel within Nigeria (from Enugu) which may be useful
+information to have for planning future workshops.
+
+![Yaé teaches
+Sage in analysis session](/public/images/SageDays102/analysis-session.jpg){:class="img-responsive"}
+
+![Olalekan teaches
+R](/public/images/SageDays102/r-session.jpg){:class="img-responsive"}
+
+This pattern seemed to be the most helpful to the most people, so for
+Thursday we repeated it: In the morning we continued the general SageMath
+and Python programming instruction from the tutorial notebooks we were
+using, and in the afternoon split into the same break-out sessions, although
+Olalekan was not able to attend Thursday, so most of his session's attendees
+split off between the other sessions, or worked on their own.
+
+Finally, on Friday, we spent a little more time following up with exercises
+that were given in the break-out sessions, following up on attendees'
+questions, and giving individual help where possible.  This continued most
+of the morning.  Just before lunch, the whole class had a 30 minute Skype
+session with Jessica Striker, an assistant professor at North Dakota State
+University, and one of the organizers of the [first Women in Sage
+workshop](https://opendreamkit.org/2017/04/06/WomenInSage/) held in Paris in
+2017, also sponsored by OpenDreamKit.  She spoke to us about that workshop
```

Comment:
maybe remove "first" in "first Women in Sage workshop held in Paris in 2017" or rephrase as "the first Women in Sage workshop to be held in the Paris area in 2017"; Women in Sage workshops listed at the Workshops page on the Sage wiki could be numbered as follows:

- Sage Days 98 -- Women in Sage 10 -- April 8 - 12, 2019 Archanes, Crete
- Sage Days 95 -- Women in Sage 9 -- July 9--13, 2018, Montréal, Canada
- Sage Days 90 -- Women in Sage 8 -- October 22-25, 2017, Harvey Mudd College, Claremont, California, USA
- Sage Days 82 -- Women in Sage 7 -- January 9-13, 2017, Paris area, France
- Sage Days 69 -- Women in Sage 6 -- September 4-9, 2015, San Diego, California, USA
- Sage Days 62 -- Women in Sage 5 -- July 28-August 1, 2014, Portland, Oregon, USA
- Sage Days 50 -- Women in Sage 4 -- July 10 - 15, 2013, Seattle, Washington, USA
- Sage Days 42 -- Women in Sage 3 -- July 15-19, 2012, Wallace Falls Lodge, Seattle, Washington, USA
- Sage Days 33 -- Women in Sage 2 -- September 19-23, 2011, Seattle, Washington, USA
- Sage Days 26 -- Women in Sage 1 -- December 7-10, 2010, Seattle, Washington, USA